Automattic/newspack-plugin
# Update various elements of the onboarding wizard

### All Submissions:






### Changes proposed in this Pull Request:

Changes are mainly around the Design Step:

* add badge for selected item
* update the preview button to button-card
* update wording

There's also a small tweak on the Services step:
* new link for the mailchimp api

### How to test the changes in this Pull Request:

1. Swithc to this branch
2. Setup Wizard
3. Services -> check the new link
4. Design -> check the preview button, and the selected state of the theme and layout

### Other information:
